lib/rubocop/rspec/language.rb

```diff
@@ -124,6 +124,10 @@ module Subject
         ALL = SelectorSet.new(%i[subject subject!])
       end
 
+      module Variables
```

Comment:
So it's basically `Helpers::ALL` + `Subject::ALL`
Not sure if we need to introduce it as a separate concept, but even if do, we can reuse the above